examples/3d/3d_scene.rs

```diff
@@ -40,3 +46,27 @@ fn setup(
             ..Default::default()
         });
 }
+
+fn follow(
+    mut q: Query<&mut Transform, With<Follow>>,
+    q_camera: Query<(&Camera, &GlobalTransform)>,
```

Comment:
You probably want to tweak this to exclude UI cameras to make it easier to port to user code.

Reply:
I haven't played with UI yet, tip for what struct exactly I should exclude?